hugegraph-core/src/main/java/com/baidu/hugegraph/auth/StandardAuthManager.java

```diff
@@ -516,10 +532,228 @@ public UserWithRole validateUser(String token) {
         return new UserWithRole(user.id(), username, this.rolePermission(user));
     }
 
+    @Override
+    public Id createProject(HugeProject project) {
+        E.checkArgument(!Strings.isNullOrEmpty(project.name()),
+                        "The name of project can't be null or empty");
+        return commit(() -> {
+            // Create project admin group
+            if (project.adminGroupId() == null) {
+                HugeGroup adminGroup = new HugeGroup(project.adminGroupName());
+                adminGroup.creator(project.creator());
```

Comment:
don't set creator here

Reply:
"creator" is a necessary parameter, if it is not passed, it will cause the creation to fail